common/defs-cmds.go

```diff
@@ -223,7 +223,7 @@ These actions are performed through subcommands for each Operator area.
 To run an Operator command, run `+"`"+`temporal operator [command] [subcommand] [command options] [arguments]`+"`"+`.
 `
 
-const ActivityUsageText = `Activity commands enable operations on [Activity Executions](/concepts/what-is-an-activity-execution).
+const ActivityUsageText = `Activity commands operate on [Activity Executions](/concepts/what-is-an-activity-execution).
```

Comment:
```suggestion
const ActivityUsageText = `Commands to operate on [Activity Executions](/concepts/what-is-an-activity-execution).
```

Make sure also that this isn't rendered in the CLI output, only for markdown.